proposals/0410-atomics.md

```diff
@@ -1051,7 +1051,7 @@ class Counter {
 
 By declaring this variable as a `var`, we opt into Swift's dynamic exclusivity checking for this property, so all non-exclusive accesses incur a runtime check to see if there is an active exclusive (e.g. mutating) access. This inherently means that atomic operations through such a variable will incur undesirable runtime overhead -- they are no longer purely atomic. (Even if the check never actually triggers a trap.)
 
-To prevent users from accidentally falling into this trap, `Atomic` (and `AtomicLazyReference`) will not support `var` bindings. It is a compile-time error to have a `var` that has an explicit or inferred type of `Atomic`.
+To prevent users from accidentally falling into this trap, `Atomic` (and `AtomicLazyReference`) will not support `var` bindings. It is a compile-time error to have a `var` that has an explicit or inferred type of `Atomic`. This behavior is gated behind the `StaticExclusiveOnly` exprimental feature flag.
```

Comment:
Mmm...going forward, the intended design isn't that this enforcement will require an experimental feature flag, so we probably shouldn't write it as such here I think. Rather, now that the proposal is accepted, one would think the behavior should (eventually) stop being gated behind a flag.

Reply:
I agree to be honest, I'd be more happy if it's simply taken out of the flag gate. Considering the attribute is an additive change, having it enabled by default doesn't come with drawbacks as far as I know.